src/xdocs/config_imports.xml

```diff
@@ -1691,24 +1743,48 @@ import java.util.stream.IntStream;
   &lt;property name=&quot;sortStaticImportsAlphabetically&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
+        <p>
+          Example:
+        </p>
+        <source>
+import static java.io.File.createTempFile;
+import static java.lang.Math.abs; // Ok, alphabetical case sensitive ASCII order, 'i' &lt; 'l'
+import java.lang.Math.sqrt; // Ok, Follow property 'Option' value 'above'
+import java.io.File; // Violation, alphabetical case sensitive ASCII order, 'i' &lt; 'l'
 
-        <p>To configure the check so that it matches default IntelliJ IDEA formatter
-           configuration (tested on v2018.2):</p>
+import java.io.IOException; // Violation; Extra separation in import group
+
+import org.albedo.*;
+
+import static javax.swing.WindowConstants.*; // Violation; Should follow 'ordered' property
+import javax.swing.JComponent;
+import org.apache.http.ClientConnectionManager; // Violation; should follow 'separated' property
+import org.linux.apache.server.SoapServer; // Ok
+
+import com.neurologic.http.HttpClient; // Ok
+import com.neurologic.http.impl.ApacheHttpClient; // Ok
+
+public class SomeClass { }
+        </source>
+        <p>
+          To configure the check so that it matches default IntelliJ IDEA formatter
+          configuration (tested on v2018.2):
+        </p>
         <ul>
           <li>group of static imports is on the bottom</li>
           <li>groups of type imports: all imports except of &quot;javax&quot;
               and &quot;java&quot;, then &quot;javax&quot; and &quot;java&quot;</li>
           <li>imports will be sorted in the groups</li>
           <li>groups are separated by, at least, one blank line and aren't separated internally</li>
         </ul>
-
         <p>
           Note: a <a href="config_filters.html#SuppressionXpathSingleFilter">
           suppression xpath single filter</a>
           is needed because IDEA has no blank line between "javax" and "java".
           ImportOrder has a limitation by design to enforce an empty line between groups
-          ("java", "javax"). There is no flexibility to enforce empty lines between some
-          groups and no empty lines between other groups.</p>
+          ("java", "javax"). There is no flexibility to enforce empty lines between
+          some groups and no empty lines between other groups.
+        </p>
```

Comment:
Please revert this change.

Reply:
if i revert this i will get something more than 100 symbols error.

Reply:
done

Reply:
Resolved.